src/Recommendations/Rule/BidirectionalRelationship.php

```diff
@@ -16,6 +16,9 @@
 use Psr\Log\LoggerInterface;
 use Throwable;
 
+/**
+ * TODO: rename into RelatedArticles?
```

Comment:
I think this is also use to relate Articles to subjects and Collections to its Articles.